test_convert.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+import utils
+
+class TestConvert(unittest.TestCase):
+
+    def test_convert(self): 
+        notebook_json = utils.read_json_file('./notebooks/test1.ipynb')
+        notebook_html = utils.perform_notebook_conversion(notebook_json)
+        self.assertIsNotNone(notebook_html)
```

Comment:
simple test, but asserts that we do indeed return something. 

Open to other ideas for verification. I did briefly consider validating the html, or saving the html locally to verify against, but im against such static tests (if we change the notebook we need to regenerate, etc). Validating HTML is a bit tricky too since no html is ever guaranteed to be valid with browsers not caring...

Reply:
Yeah, i thought about that too, but I've been bitten more times I can count from asserting raw string values from third party output, and then something changes on their end and I need to stop what I'm doing and look at the build infrastructure.

Reply:
Fair enough. I was thinking you could check for a string that is in the notebook file, but that's certainly also not guaranteed to come across without some sort of modification. I leave the decision to you.